server/lib/src/server/mod.rs

```diff
@@ -1500,8 +1510,20 @@ impl<'a> QueryServerWriteTransaction<'a> {
     pub(crate) fn reload_domain_info(&mut self) -> Result<(), OperationError> {
         let domain_name = self.get_db_domain_name()?;
         let display_name = self.get_db_domain_display_name()?;
+        let domain_ldap_allow_unix_pw_bind = match self.get_domain_ldap_allow_unix_pw_bind(){
+            Ok(v) => v,
+            _ => {
+                admin_warn!("Defaulting ldap_allow_unix_pw_bind to false");
+                self.set_domain_ldap_allow_unix_pw_bind(false)?;
+                false
```

Comment:
For now we have to default it to true here. This is because if we defaulted it to false right now, we'd break ldap for a lot of people.